### DISCUSSION TITLE
Is there a way to send a webhook?

### DISCUSSION THREAD

**Discussion by havedill** [Budibase/budibase#4971] [Help]
**Describe the feature request**
Basically, i'd like the application i'm developing to have the ability to trigger a webhook on some of our other services. Jenkins for example, and if we adopt it Apache Airflow

Example:
Support team selects some service values from an API pull
They trigger the submit button
Action item sends a webhook to 3rd party app, which does a workflow to restart that service

I was trying to use the bash automation for it, but tools like wget or curl are not available to me.

**Comment by mjashanks** [comment]
You can use a webhook. You will need to

1. In the `Data` section, setup a REST API: https://docs.budibase.com/docs/rest
2. Add an automation step to `External data connector`
3. Link it to your REST query
